### DISCUSSION TITLE
Atoms experience attractive forces at short distances

### DISCUSSION THREAD

**Discussion by AditiKhot** [deepmodeling/deepmd-kit#45] [Q&A] [answered]
Hi,

I am trying to use DeePMD package for DeePCG according to the[ paper.](https://aip.scitation.org/doi/abs/10.1063/1.5027645) I am having some issues in the training procedure. Right now, when I run LAMMPS with the trained forces, clusters of atoms come very close ~0.5 A. It looks like the training is not sampling the repulsive core region correctly and the forces are attractive forces at short distances. 

I am using a trajectory of 15ns with sampling at 100fs, which is equivalent to 150000 sample for training.  The  neural network has the same structure as that from the paper. And the values of batch_size, start_lr, decay_steps and decay_rate are 4, 0.001, 5000 and 0.95 respectively.
Please let me know what I could do to fix this issue.

Thank you very much! 
Aditi.

**Comment by amcadmus** [comment, marked as answer]
Hi Aditi,

We came up with similar issue when the training is not fully converged. I would recommend you train you system with more "stop_batch". It is noted that the "decay_steps" should increase correspondingly to make sure that the stop learning rate is of order 1e-8. 

It is possible to check the quality of the model by the model deviation, as mentioned in section IV of the paper. It lets you know when your model fails in predicting the cg forces of a configuration.
